internal/service/domain/tree/tree.go

```diff
@@ -106,10 +105,11 @@ func (s *TreeService) Create(ctx context.Context, treeCreate *entities.TreeCreat
 		return nil, handleError(err)
 	}
 
-	if err = s.locator.UpdateCluster(ctx, treeCreate.TreeClusterID); err != nil {
-		return nil, handleError(err)
+	if treeCreate.TreeClusterID != nil {
```

Comment:
It should be possible to create a tree without a treecluster!

Reply:
Yes, where is the problem? :)